### ISSUE TITLE
Admin\Contacts - External User can view a contact

### ISSUE THREAD

**Issue by zoeyli-46** [bcgov/cas-registration#1710]
#### Description:
As an external user I want to see the details for each contact 

#### Acceptance Criteria:
Given I am on the contacts grid #1350 
When I click on view details
Then I am brought to the contact details page in view mode
[Figma](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=18028-7380&t=ekoHMMxeXQOu4QFM-4)

I can see the following places that a contact is assigned to:
Operation Representative, and Person Responsible for Preparing and Submitting a Report (once they've been added via reporting)

#### Development Checklist:

- [x] `View Details` functionality implemented in #1607
- [x] Add field `Places Assigned`
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies

- Blocked by
- Blocking

**Comment by Sepehr-Sobhani** [comment]
This is addressed in: https://github.com/bcgov/cas-registration/pull/1940 **EXCEPT** `Places Assigned` field

**Comment by BCerki** [comment]
Dev work is well on it's way, but we need to know which places assigned to show before we can finish it cc @zoeyli-46

**Comment by BCerki** [comment]
To test this, you can register an operation, add operation representatives, and then go look at those contacts and you should see them listed as reps for the operation you assigned them to. Alternatively, Henry Ives is set up as an operation rep for Operation 2 in our mock data. 

If you look at anyone who isn't assigned anywhere, you'll see "None" instead of a list of places. (This wasn't in the designs/AC, we guessed, if not desired we can change.)

Note: Person Responsible for Preparing and Submitting a Report doesn't exist yet, so at the moment, you'll only see operation reps.

**Comment by patriciarussellCAS** [comment]
Thanks @BCerki - this looks good in Dev. I agree that adding "none" makes sense in places assigned.
